plugins/l4d2/l4d2.pro

```diff
@@ -6,5 +6,15 @@
 include(../plugins.pri)
 
 TARGET		= l4d2
-SOURCES		= l4d2.cpp
-LIBS		+= -luser32
+
```

Comment:
Can we please move the plugin to `plugins/l4d2-linux` instead? So, in its own directory, with its own .pro file.